ni_measurementlink_service/measurement/info.py

```diff
@@ -11,7 +11,7 @@ class MeasurementInfo(NamedTuple):
 
     Attributes
     ----------
-        display_name (str): The measurement display name for client to display to user.
+        display_name (str): The measurement display name for clients to display to users.
 
         version (str): The measurement version that helps to
         maintain versions of a measurement in future.
```

Comment:
'in the future'

Reply:
https://github.com/ni/measurementlink-python/pull/378